providers/implementations/macs/kmac_prov.c.in

```diff
@@ -389,53 +392,55 @@ static int kmac_final(void *vmacctx, unsigned char *out, size_t *outl,
     return ok;
 }
 
-static const OSSL_PARAM known_gettable_ctx_params[] = {
-    OSSL_PARAM_size_t(OSSL_MAC_PARAM_SIZE, NULL),
-    OSSL_PARAM_size_t(OSSL_MAC_PARAM_BLOCK_SIZE, NULL),
-    OSSL_FIPS_IND_GETTABLE_CTX_PARAM()
-    OSSL_PARAM_END
-};
+{- produce_param_decoder('kmac_get_ctx_params',
+                         (['MAC_PARAM_SIZE',                    'size',   'size_t'],
```

Comment:
multiple spaces here..

Reply:
???? where ????

I'm just aligning the columns with those below and following the same format I've used elsewhere.